config/config.go

```diff
@@ -112,7 +112,7 @@ var (
 			MaxNumActsPerPool:  32000,
 			MaxGasLimitPerPool: 320000000,
 			MaxNumActsPerAcct:  2000,
-			ActionExpiry:       10 * time.Minute,
+			ActionExpiry:       3 * time.Minute,
```

Comment:
with this PR, transfer/exec with bad address will remain in actpool

thinking to reduce the expire time, to speed up removing it out of actpool

Reply:
Don't integrate this change in this pr

Reply:
yes, we can discuss this, and another PR if needed